source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -508,6 +508,18 @@ def __init__(self):
                                      "actually a minimum.\n"
                                      "Nota 2: 'max_nb_journeys' parameter has priority over "
                                      "'timeframe_duration' parameter.")
+        parser_get.add_argument("max_nb_crowfly_by_walking", type=int,
+                                help="limit nb of stop points accesible by crowfly, "
+                                     "used especially in distributed scenario")
+        parser_get.add_argument("max_nb_crowfly_by_car", type=int,
+                                help="limit nb of stop points accesible by crowfly, "
+                                     "used especially in distributed scenario")
+        parser_get.add_argument("max_nb_crowfly_by_bike", type=int,
```

Comment:
I did not think it is usefull to add it to /journeys (the JIRA is not clear on that, and we each thought of different things during refinement it seems :-/ ).
What you did could be easier to check impacts and negociate with partners, but it would not be permanent code (only debug/test) and I think it would be better to have a param only in tyr so noone can reactivate bike calls if we "deactivate" it in tyr.

Same as @kinnou02 at least, these should not be public params (`_` and `hidden=True`)